test/kata6.joinNames.test.js

```diff
@@ -1,5 +1,41 @@
 const { joinNames } = require("../src");
 
 describe("joinNames", () => {
-  test("returns string of names, seperated by commas and an ampersand", () => {});
+  // eslint-disable-next-line prettier/prettier
```

Comment:
Why disable prettier here? I would remove all these and let the formatter do its job